README.md

```diff
@@ -1,13 +1,133 @@
-<h1> imageCarousel </h1>
-<h3> How to run </h3>
-<li> npm run seedDB </li>
-<li> npm run react-dev </li>
-<li> npm run start </li>
-
-<h1> Endpoints </h1>
-<li> GET /houses/:houseId/ </li>
-<li> GET /houses/ </li>
-<li> POST /houses/ </li>
-<li> PUT /houses/:houseId/ </li>
-<li> DELETE /houses/:houseId/ </li>
+# Getting started with Image Gallery:
 
+1) npm run seedDB
+2) npm run react-dev
+3) npm run start
+
+# CRUD API Documentation# Image Gallery
+
+### Listing Routes:
+
+* **URL**: /listings/:listingID/images
+  * **Notes:** Gets data for all images in a specific listing
+  * **Method:** _`GET`_
+  * **Example:** `GET listings/25/images`   
+  Gets all data from images in a listing with ID 25
+  * **URL Params:** `listingID = [integer]`
+  * **Returns:**
+```javascript
+image = [
+  { 
+    photo_url VARCHAR(250) NOT NULL,
+    listing_address VARCHAR(250) NOT NULL,
+    price MONEY NOT NULL,
+    numBedrooms INTEGER NOT NULL,
+    numBathrooms INTEGER NOT NULL,
+    sqft INTEGER NOT NULL, 
+    saleStatus VARCHAR(250) NOT NULL,
+    saved BOOLEAN NOT NULL
+  }
+]
```

Comment:
Great detail: I found this very easy to follow.  The METHOD and URL paarams is a nice touch an d something that I am going to add to my route documentation